utensor_cgen/code_generator.py

```diff
@@ -20,24 +19,35 @@
 
 
 class CodeGenerator(object):
-  def __init__(self, pb_file, idx_dir, embed_data_dir, method, output_nodes, 
+  def __init__(self, model_file,
+               idx_dir,
+               embed_data_dir,
+               methods,
+               output_nodes,
                debug_cmt=False,
-               clear_tmp_file=True):
-    self.pb_file = pb_file
+               **trans_kwargs):
+    self.model_file = model_file
     if not os.path.exists(idx_dir):
       os.makedirs(idx_dir)
     self.idx_dir = idx_dir
     self.embed_data_dir = embed_data_dir.rstrip("/")
-    self.method = method
+    self.methods = methods
```

Comment:
Um... the naming is not good honestly.
`methods` means the transformation methods you want to apply to your graph.
This will be used to setup a transformation pipeline later.

Reply:
Let's change it to `transform_methods` then for now